attachment_queue/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module implements a queue for processing file.
+The files are stored in Odoo standard ir.attachment.
+The attachments will be processed depending on their type.
```

Comment:
```suggestion
This module adds async processing capabilities to attachments by implementing a new model attachment.queue (suggest: attachment.async.process) that wraps attachments and stores additional information so that it can be processed in an asynchronous way.
```